src/doc/src/reference/unstable.md

```diff
@@ -112,6 +112,8 @@ Each new feature described below should explain how to use it.
 * Registries
     * [publish-timeout](#publish-timeout) --- Controls the timeout between uploading the crate and being available in the index
     * [asymmetric-token](#asymmetric-token) --- Adds support for authentication tokens using asymmetric cryptography (`cargo:paseto` provider).
+* Mafinest and lockfile 
```

Comment:
Not sure if we should add another section, but existing ones don't seem to be relative enough for this feature, WDYT?

Reply:
Maybe under Other suffices